annotation/feedback_default_options.txt

```diff
@@ -0,0 +1,9 @@
+shortname,category,description
```

Comment:
Nitpick: rename this .csv since it's comma-separated.

Reply:
done!